### DISCUSSION TITLE
注册账号时显示错误

### DISCUSSION THREAD

**Discussion by alexjjzc** [walinejs/waline#1077] [Q&A] [answered]
注册账号时，显示“注册确认邮件发送失败，请确认你的邮箱输入无误并联系管理员。”该如何解决！
![image](https://user-images.githubusercontent.com/61019155/171098432-9fc35c96-fcb9-48bb-8efb-c16d5dbbd73d.png)

**Comment by lizheming** [comment, marked as answer]
去 Vercel 上检查一下邮件发送的日志，确认下邮件发送是否正常。

**Reply by alexjjzc** [reply]
不好意思，我可能没说清楚，是Waline注册账号时显示这个错误信息

**Reply by lizheming** [reply]
已处理，感谢反馈
